packages/gatsby/src/schema/schema.js

```diff
@@ -153,6 +153,11 @@ const processTypeComposer = async ({
       fieldExtensions,
       parentSpan,
     })
+    await determineSearchableFields({
```

Comment:
We mark fields that are searchable (so should be included in filter), sortable (so should be included in sort) and resolvable (so should be prepared) via extensions metadata.